commerce_bluesnap.module

```diff
@@ -5,6 +5,8 @@
  * Provides Commerce integration with the BlueSnap Payment Gateway.
  */
 
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\commerce_payment\Entity\PaymentMethodInterface;
```

Comment:
**CHANGE REQUEST**

These do not seem to be used. Get into the habit to run the code through PHPCS before committing, it will warn you for such mistakes.

Reply:
Oops, My bad.Removed